frontend/src/metabase/api/dashboard.ts

```diff
@@ -109,10 +109,7 @@ export const dashboardApi = Api.injectEndpoints({
       invalidatesTags: (_, error) =>
         invalidateTags(error, [listTag("dashboard")]),
     }),
-    listEmbeddableDashboards: builder.query<
-      GetPublicOrEmbeddableDashboard[],
-      void
-    >({
+    listEmbeddableDashboards: builder.query<GetPublicDashboard[], void>({
```

Comment:
Should this be `GetEmbeddableDashboard` instead of `GetPublicDashboard`?

Reply:
yes!